python/res/enkf/jobs/enkf_scaling_get_data.py

```diff
@@ -0,0 +1,173 @@
+from copy import deepcopy
```

Comment:
Should be split up into `enkf_scaling_measured_data.py` (or something) and `enkf_scaling_scaled_matrix.py` (or something).

Reply:
Maybe move everything into `enkf/jobs/scaling`?

Reply:
:+1: